src/components/FullPlayground.tsx

```diff
@@ -361,29 +353,26 @@ interface SharingState {
   shareReference?: string;
 }
 
-export function FullPlayground(props: { withRouter?: any }) {
+export function FullPlayground() {
   return  <>
       <DiscordChatCrate
         serverId={AppConfig().discord.serverId}
         channelId={AppConfig().discord.channelId}
       />
-      <ApolloedPlayground withRouter={props.withRouter} />
+      <ApolloedPlayground />
     </>
 }
 
-function ApolloedPlayground(props: { withRouter?: any }) {
+function ApolloedPlayground() {
   const datastore = usePlaygroundDatastore();
-  const Router = props.withRouter ? props.withRouter : Box;
   return (
-    <Router>
```

Comment:
This got lifted up, which means we could get rid of the props and this wrapping.